<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;NEXUS-01 :: PANEL DE TRÁFICO AEROESPACIAL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>NAVE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>131</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>OPERACION</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>131</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>FECHA/HORA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>131</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>ORIGEN/DESTINO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>131</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>BITACORA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_registrar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>REGISTRAR TRÁNSITO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_borrar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>500</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>BORRAR EVENTO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_modificar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>250</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>MODIFICAR ENTRADA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_nave">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_destino">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dt_fecha">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="box_operacion">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tb_vuelos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>791</width>
      <height>261</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>791</width>
      <height>16777215</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>NAVE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>OPERACION</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>FECHA/HORA</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ORIGEN/DESTINO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>BITACORA</string>
     </property>
    </column>
   </widget>
   <widget class="QTextEdit" name="txt_bitacora">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_guardar">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>270</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>GUARDAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
